docs/guides/js-waku/use-waku-react.md

```diff
@@ -0,0 +1,356 @@
+---
+title: "Build React DApps Using @waku/react"
+---
+
+The [@waku/react](https://www.npmjs.com/package/@waku/react) package provides components and UI adapters to integrate `js-waku` into React applications effortlessly. This guide provides detailed steps for using `@waku/react` in your project.
+
+## Install the Dependencies
+
+First, set up a project using any [production-grade React framework](https://react.dev/learn/start-a-new-react-project) or use an existing React application. For this guide, we will create a boilerplate using [Create React App (CRA)](https://create-react-app.dev/docs/getting-started):
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+<Tabs groupId="package-manager">
+<TabItem value="npm" label="npm">
+
+```shell
+npx create-react-app [PROJECT DIRECTORY]
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```shell
+yarn create react-app [PROJECT DIRECTORY]
+```
+
+</TabItem>
+</Tabs>
+
+Next, install the `@waku/react` package using your preferred package manager:
+
+<Tabs groupId="package-manager">
+<TabItem value="npm" label="npm">
+
+```shell
+npm install @waku/react @waku/sdk
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```shell
+yarn add @waku/react @waku/sdk
+```
+
+</TabItem>
+</Tabs>
+
+## Create a Relay Node
```

Comment:
Let's remove relay node from all docs. It's pointless to use it at this point in time.